README.Rmd

```diff
@@ -0,0 +1,66 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# hydromad
+
+<!-- badges: start -->
```

Comment:
Can apply [badgecreatr](https://cran.r-project.org/web/packages/badgecreatr/readme/README.html) if you like... May make more sense after we have #53 merged

Reply:
Indeed, I was waiting for that, which is why I left it commented out